03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html

```diff
@@ -0,0 +1,18 @@
+<!-- Code generated by market-hour-convertor.py -->
+<p>The following table shows the available contracts in the OANDA Forex market. Some of these contracts may have different trading periods than the overall OANDA Forex market.</p>
+
+<table class="table qc-table table-reflow">
+<thead>
+<tr><th>Symbol</th><th>Contract</th></tr>
+</thead>
+<tbody>
+<tr><td><a href="oanda/eurnzd">EURNZD</a></td><td>EUR/NZD</td></tr>
```

Comment:
Let's replace this table for a list. Both of these column pretty much contain the same information

Reply:
I replied in the next comment